src/js/control-bar/progress-control/load-progress-bar.js

```diff
@@ -61,16 +61,26 @@ class LoadProgressBar extends Component {
     const children = this.partEls_;
 
     // get the percent width of a time compared to the total end
-    const percentify = function(time, end) {
+    const percentify = function(time, end, rounded) {
       // no NaN
-      const percent = (time / end) || 0;
+      let percent = (time / end) || 0;
 
-      return ((percent >= 1 ? 1 : percent) * 100) + '%';
+      percent = (percent >= 1 ? 1 : percent) * 100;
+
+      if (rounded) {
+        percent = percent.toFixed(2);
+      }
+
+      return percent + '%';
     };
 
     // update the width of the progress bar
     this.el_.style.width = percentify(bufferedEnd, duration);
 
+    // update the control-text
+    // TODO: Need a better way to reference the node that needs its textContent set
```

Comment:
I'm looking for suggestions on how to implement this.

Reply:
Alternatively, make a helper function that given a percentage, return the innerHTML text, something like:
```js
const buildEl = (percent) => `<span class="vjs-control-text"><span>${this.localize('Loaded')}</span>: ${percent}</span>`